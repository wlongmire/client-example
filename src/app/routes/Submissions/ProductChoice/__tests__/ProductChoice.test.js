/* eslint-env node, jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { ProductChoice } from '../index'

const submission = {
  type: 'oi',
  clearanceStatus: 'pass'
}

describe('>>> ProductChoice Component --- Snapshot', () => {
  it('+++ product choice page', () => {
    const renderedValue = shallow(
      <ProductChoice
        user={{ broker: 'test-2222-7fd-b3ff-4fd3-9fc2-e752b9f5b002'}}
      />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})