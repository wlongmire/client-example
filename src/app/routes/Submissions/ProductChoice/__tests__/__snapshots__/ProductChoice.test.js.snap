// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> ProductChoice Component --- Snapshot +++ product choice page 1`] = `
<div
  className="productChoice routeContainer"
>
  <h3>
    Select Your Insurance Product.
  </h3>
  <div
    className="selectionCards"
  >
    <Connect(ProductChoiceItem)
      description="Owners of construction projects should have their own liability coverage. The Owner's Interest policy closes the gaps in the owner's liability insurance program and assure more adequate protection."
      name="Owner's Interest"
      type="oi"
    />
    <Connect(ProductChoiceItem)
      description="The Owners and Contractors Protective policy responds to liability arising out of the insured's own acts or omissions in connection with its general supervision of the contractor's operations."
      name="Owners and Contractors Protective"
      type="ocp"
    />
  </div>
  <LinkContainer
    action="push"
    onlyActiveOnIndex={false}
    to="/submissions"
  >
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="default"
      className="btn"
      disabled={false}
    >
      Return to Submissions
    </Button>
  </LinkContainer>
</div>
`;
